applicationinsights-js/index.d.ts

```diff
@@ -717,16 +721,16 @@ declare module Microsoft.ApplicationInsights {
             [name: string]: number;
         }): any;
         /**
-        * Log an AJAX request
-        * @param  id  Event id
-        * @param  absoluteUrl Full url
-        * @param  pathName Leave this parameter blank
-        * @param  totalTime Total time it took for AJAX request to complete
-        * @param  success Whether AJAX request succeeded or failed
-        * @param  resultCode Result code returned from AJAX call
-        * @param  method  HTTP verb that was used (GET, POST)
-        */
-        trackAjax(id: string, absoluteUrl: string, pathName: string, totalTime: number, success: boolean, resultCode: number, method?: string): any;
+         * Log a dependency call
+         * @param id    unique id, this is used by the backend o correlate server requests. Use Util.newId() to generate a unique Id.
```

Comment:
Can you align the descriptions in some way (either one space between param name and description or have all descriptions start at the same column)?